src/main/java/fr/skyost/owngarden/worldedit/Utils.java

```diff
@@ -0,0 +1,65 @@
+package fr.skyost.owngarden.worldedit;
+
+import fr.skyost.owngarden.OwnGarden;
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.text.format.NamedTextColor;
+import org.bukkit.Bukkit;
+import org.bukkit.Location;
+import org.bukkit.plugin.Plugin;
+
+import java.nio.file.Path;
+
+public interface Utils {
+
+    /*
+     * Accepted WorldEdit versions.
+     */
+//    String[] FAWE_VERSIONS = {"2.9.1", "2.9.2"};
+
+    /**
+     * Returns whether the current WorldEdit version should be accepted.
+     *
+     * @return Whether the current WorldEdit version should be accepted.
+     */
+
+    static boolean checkWorldEditVersion(final OwnGarden plugin) {
+        final Plugin we = Bukkit.getPluginManager().getPlugin("FastAsyncWorldEdit");
```

Comment:
Just check if "WorldEdit" here. Because FAWE provide `WorldEdit` via Provider field over the plugin.yml